src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -33,12 +34,14 @@ case class HeaderCarrier(authorization: Option[Authorization] = None,
                          requestId: Option[RequestId] = None,
                          requestChain: RequestChain = RequestChain.init,
                          nsStamp: Long = System.nanoTime(),
-                         extraHeaders: Seq[(String, String)] = Seq(), 
+                         extraHeaders: Seq[(String, String)] = Seq(),
```

Comment:
Hmmmm, this seems suspiciously similar to 'otherHeaders'... I wonder what it's used for